src/statement.cc

```diff
@@ -856,7 +856,9 @@ void Statement::Finalize(Baton* baton) {
     // Fire callback in case there was one.
     Napi::Function cb = baton->callback.Value();
     if (!cb.IsUndefined() && cb.IsFunction()) {
-        TRY_CATCH_CALL(baton->stmt->Value(), cb, 0, NULL);
+        //TRY_CATCH_CALL(baton->stmt->Value(), cb, 0, NULL);
```

Comment:
I'm wondering why TRY_CATCH_CALL does not work in this case, and if its something we can fix in the macro.  I'm assuming that the TRY_CATCH_CALL may have been from the original code and if so keeping the port aligned would be good.